models/edr/run_results/query_history.sql

```diff
@@ -0,0 +1,19 @@
+{{
```

Comment:
How do we ensure that this model doesn't error for users who don't have permissions to `JOBS`?
Do everyone have the option to query it and at least get partial results (only for their user, etc.)?

Reply:
the permission to list jobs is pretty basic in bigquery so i dont think we will encounter it, but i can (and will now) add a `can_query_relation` check